examples/minimal-bun-react/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "@examples/bun-react",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "workspaces": [
+    "client",
+    "server"
+  ],
+  "scripts": {
+    "dev": "concurrently \"bun --cwd client dev\" \"bun --cwd server dev\"",
+    "FIXME": "Something seems off with build script in our monorepo. It works standalone...",
+    "_build": "bun --cwd client build && bun --cwd server build",
```

Comment:
idk what's up here... i can build locally just fine and if you pull it out and do it there it works as well... something happens when CI tries to built it inside the monorepo 🤔